ui/components/requirements.jsx

```diff
@@ -11,16 +11,18 @@ function Requirement({ requirement }) {
   return (
     <li className="req border rounded p2 mb2 clearfix max-width-3">
       <div className="req-id col col-1 mb2">
-        {requirement.req_id}
+        { requirement.req_id }
       </div>
-      <div className="req-text col col-11">
-        {requirement.req_text}
+      <div className="req-text col col-11 pl1">
+        { requirement.req_text.split('\n').map(line => (
+          <span className="req-text-line mb1">{ line }<br /></span>
+          ))}
```

Comment:
Nice solution! It might make sense to pull this out into a utility function if we end up printing the text out in more places.